docs/guides/user-guide/available-environments.md

```diff
@@ -56,3 +56,22 @@ This environment has a terminal that supports various common commands you would
 WebContainers are currently supported by Chromium-based browsers and [in Firefox, with some caveats](/platform/webcontainers/browser-support).
 
 Our [Codeflow IDE](/codeflow/what-is-codeflow) runs on the WebContainers runtime.
+
+# Determining Which Environment You Are Using
+
+:::tip Not seeing a terminal in your project?
+
+Currently, the terminal is only available in projects which use the WebContainers execution environment. There is no way to automatically convert from one execution environment to another.
```

Comment:
Don't tell them they can't autop convert without providing the workaround. Probably best ot just not mention it at all